app/graphql/resolvers/categories_search.rb

```diff
@@ -42,6 +43,10 @@ def apply_order(scope, value)
     scope.order(value)
   end
 
+  def apply_location(scope, _value)
+    scope
+  end
```

Comment:
does anyone know another possibility to apply an option? if i do not add the `with`-function, a `where` is tried on `location` automatically, which does not exist on categories. so i decided to give this "just return scope" method. this could not be best practice :/!?

Reply:
I don't understand what this does here. Just calling scope won't alter the results, right? So why have the option then?

Reply:
right, it does nothing. but i haven't found another solution to be able to pass a `location` option, that i can access in the field resolvers "later"

Reply:
Man kann auch den einzelnen Fields scheinbar argument übergeben.
`query { categories{name genericItemsCount(location: "Hamburg")} }`

ich probiere das in einem Commit umzusetzen

Reply:
this can be removed now after update with #315 as we do not need a location option here anymore

Reply:
done with dcaef74